### ISSUE TITLE
Reproduce DexYCB results

### ISSUE THREAD

**Issue by ZhenyueQin** [namepllet/HandOccNet#16]
Hi, 

Thank you for producing such fantastic work for the community. I followed the training instructions and trained a model of DexYCB for 25 epochs. I got very poor results: 

MPJPE : 55.41 mm
PA MPJPE : 14.15 mm

Can the authors please shed light on where I may have gone astray? 

Thank you very much!

**Comment by namepllet** [comment]
Could you capture your train_logs.txt (saved in output/log) and share it ?

And did you get the same results with ours by using pretrained model ?

**Comment by ZhenyueQin** [comment]
The pretrained model's results are much better: 
- MPJPE: 10.01 mm
- PA MPJPE: 4.52 mm
You can get my train_logs.txt through this link: https://drive.google.com/file/d/1c_2qpufOuWGM_KmljYQpiXOm2em2iDZx/view?usp=sharing

**Comment by namepllet** [comment]
It seems you used GPUs less than 4.

We used 4 GPUs, so you should scale learning rate according to your batch size.

**Comment by ZhenyueQin** [comment]
Thank you :-) Let me give a try.

**Comment by LiaoQi98** [comment]
> Thank you :-) Let me give a try.

Hi, I also have the same issue. Did you achieve better results after adjusting the learning rate and batch size?
